InSimDotNet/Packets/PacketType.cs

```diff
@@ -298,6 +298,11 @@ public enum PacketType {
         /// </summary>
         ISP_JRR,
 
+        /// <summary>
+        /// get mods
+        /// </summary>
+        ISP_MAL,
+

```

Comment:
This is wrongly placed. It should be added after ISP_CIM.

It breaks following packets:
- ISP_UCO
- ISP_OCO
- ISP_TCC
- ISP_SLC
- ISP_CSC
- ISP_CIM